### PR TITLE
Adding Interesting Security Sessions

### LINKED ISSUE #1: Create a security specific agenda for Devcon4
The proposed, mostly final Devcon4 agenda is here: https://docs.google.com/spreadsheets/d/e/2PACX-1vTmQ1maZLMDSo3r7wVCzwMadNUCGctmE5byRgv1za6R52wTUgZw-XB9P9dNO7-QBRka1AAwKrXO4kTP/pubhtml#

## The task

I would like you to comb through the agenda, and identify all the sessions that would be interesting to the readers of my Newsletter. Let's include anything in the following topics:

1. cryptography
2. client security
3. smart contract security
4. EVM specific
5. Formal verifcation, testing and static analysis
6. Secure key storage
7. Gas optimization
8. on chain privacy

## The output

A PR that fleshes out this file: https://github.com/maurelian/Newsletter/blob/master/Devcon4_Security_Agenda.md

Please also do what you can within reason to add any satellite events you might know of.

### PR DESCRIPTION
**Pull request by nanspro**
Fixes #1

### CODE REVIEW

**Comment by nanspro** [comment]
@maurelian  can you review ?

File: Devcon4_Security_Agenda.md [hunk 1]
```diff
@@ -2,19 +2,72 @@
...
+	- Patrick Ventuzelo, 3:00PM - 3:25PM, Breakout Room
 
+- P2P Networking in Ethereum 2.0
+	- Jannik Luhn, Kevin Mai-Hsuan Chia, 1:30PM - 1:55PM, Side Stage
+
+- MakerDAO: A New Hope for Financial Reform
+	- Andy Milenius, 2PM - 2:30PM, Auditorium
+
+- Human Centered Identity: We are more than keys
+	- Jonny Howle, 4:35PM - 5:00PM, Side Stage
```
Comment (maurelian) [inline Devcon4_Security_Agenda.md:50]: At 4:55 pm on this day is a session called "Current State of Security Panel", so it appears that this wasn't done very thoroughly.

**Review by maurelian — APPROVED** [review]
I am going to pull this and close out the bounty, but I'm not confident that it was completely particularly thoroughly, so I'll run with what you've provided, and complete the rest myself.

File: Devcon4_Security_Agenda.md (hunk 1 above)
Comment (nanspro) [reply Devcon4_Security_Agenda.md:50]: i tried to keep around 4 events per day and i didn't get the topic properly but i went through it thoroughly so, sorry for this part !!

**Pull request merged**